app/dashboard/utils.py

```diff
@@ -824,3 +828,57 @@ def get_nonce(network, address):
     JSONStore.objects.create(key=key, view=view, data=[new_nonce])
 
     return new_nonce
+
+
+def re_market_bounty(bounty, auto_save = True):
+    remarketed_count = bounty.remarketed_count
+    if remarketed_count < 2:
+        now = timezone.now()
+        one_hour_after_remarketing = bounty.last_remarketed + timezone.timedelta(hours=1) if bounty.last_remarketed else now
```

Comment:
Lets take the remarketing time limit and maximum count out of the function and have them get set as environment variables through settings.py - this way we can easily adjust them if we need to in the future, and we won't have any kind of "magic number" confusion: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
Hey @danlipert I’ve made the changes you’ve requested😊Thanks, Vincent!